src/rust/en.reaperscans/src/request_helper.rs

```diff
@@ -0,0 +1,194 @@
+use aidoku::{
+	error::AidokuError,
+	helpers::substring::Substring,
+	prelude::format,
+	std::html::Node,
+	std::net::{HttpMethod, Request},
+	std::{String, Vec},
+};
+
+use crate::helper::USER_AGENT;
+
+pub fn create_search_request_object(
+	base_url: String,
+	search_query: String,
+) -> Result<Node, AidokuError> {
+	let html = Request::new(base_url, HttpMethod::Get)
+		.html()
+		.expect("Failed to get html");
```

Comment:
better to use `?` here so the source doesn't break when user is offline, but oh well